src/Uno.UI/UI/Xaml/Media/IGradientBrush.wasm.cs

```diff
@@ -0,0 +1,10 @@
+﻿using Windows.Foundation;
+
+namespace Windows.UI.Xaml.Media
+{
+    partial interface IGradientBrush
+    {
+		string ToCssString(Size size);
+		UIElement ToSvgElement();
+	}
+}
```

Comment:
Please use tabs and better alignments ;-)

Reply:
@carldebilly That's Visual Studio not respecting .editorconfig 😄 Will fix.

Reply:
Fixed :)